src/subscriptions.c

```diff
@@ -84,7 +84,9 @@ void read_subscriptions(struct list **subs)
 	dir = opendir(path);
 	if (dir) {
 		while ((ent = readdir(dir))) {
-			if ((strcmp(ent->d_name, ".") == 0) || (strcmp(ent->d_name, "..") == 0)) {
+			if ((strcmp(ent->d_name, ".") == 0) ||
```

Comment:
could probably just ignore everything starting with `.` here too.

Reply:
Yes. As agreed in IRC we will ignore all "dot" files, there are a few other places as well where we have this kind of logic, but didn't specifically affect adding a ".MoM" file.